<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainGraph</class>
 <widget class="QMainWindow" name="MainGraph">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>939</width>
    <height>576</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainGraph</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="appLayout" stretch="5">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="sourceLayout">
        <property name="leftMargin">
         <number>6</number>
        </property>
        <property name="topMargin">
         <number>6</number>
        </property>
        <property name="rightMargin">
         <number>6</number>
        </property>
        <property name="bottomMargin">
         <number>6</number>
        </property>
        <item>
         <widget class="QLineEdit" name="sourcePath">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string extracomment="Path to .csv file"/>
          </property>
          <property name="placeholderText">
           <string>Path to .csv file</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="findSource">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>30</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="saveLayout">
        <property name="leftMargin">
         <number>6</number>
        </property>
        <property name="topMargin">
         <number>6</number>
        </property>
        <property name="rightMargin">
         <number>6</number>
        </property>
        <property name="bottomMargin">
         <number>6</number>
        </property>
        <item>
         <widget class="QLineEdit" name="saveFilePath">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string extracomment="Save file location"/>
          </property>
          <property name="cursorPosition">
           <number>0</number>
          </property>
          <property name="placeholderText">
           <string extracomment="Save file location">Save file location</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="findSaveLocation">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="maximumSize">
           <size>
            <width>30</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="Save">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="maximumSize">
           <size>
            <width>50</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Save</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QListWidget" name="lblKeyList">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="ev">
        <property name="text">
         <string>Waiting event....</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="lblLayout" stretch="0,2,1">
        <item>
         <widget class="QLineEdit" name="lbl">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string extracomment="label"/>
          </property>
          <property name="placeholderText">
           <string>label</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="key">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string extracomment="key"/>
          </property>
          <property name="placeholderText">
           <string extracomment="key">key</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="addLbl">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>40</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>add</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>939</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
